packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -250,7 +248,6 @@ function commitBeforeMutationLifeCycles(
     case ForwardRef:
     case SimpleMemoComponent:
     case Chunk: {
-      commitHookEffectList(UnmountSnapshot, NoHookEffect, finishedWork);
```

Comment:
We don't have a snapshot phase hook (yet) so this call was walking the effect list for no reason. I've removed it.